tests/Builder/FormContractorTest.php

```diff
@@ -57,29 +61,13 @@ public function testDefaultOptionsForSonataFormTypes()
         $fieldDescription->method('getAssociationAdmin')->willReturn($admin);
 
         $modelTypes = [
-            'sonata_type_model',
-            'sonata_type_model_list',
+            ModelType::class,
+            ModelListType::class,
         ];
-        $adminTypes = ['sonata_type_admin'];
-        $collectionTypes = ['sonata_type_collection'];
-        // NEXT_MAJOR: Use only FQCNs when dropping support for Symfony <2.8
-        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
```

Comment:
This is a part of your commit you could keep

Reply:
Wait in fact, no, I think it can be removed entirely

Reply:
Sorry I misunderstood the comment, so I guess it's better if I just remove the last commit from this PR and make another one to `master` removing all this code.